### DISCUSSION TITLE
presidio performance (papers, benchmarks, comparisons, datasets, etc)

### DISCUSSION THREAD

**Discussion by gustavz** [microsoft/presidio#1226] [General]
Hi presidio team, first of all I wanted to say that I really like `presidio` (@omri374 )!

To better understand its capabilities I would be highly interested in any available information on how presidio performs with regard to the common PII detection metrics. Are there any papers, benchmark results or comparisons to other methods available?
Any resource would be highly appreciated.
Thanks!

**Comment by omri374** [comment]
Hi @gustavz, thanks for the kind words!
Unfortunately, we don't any formal results, and it's somewhat intentional. We see presidio as a framework and not a solution, and every solution created with Presidio is different and is highly dependent on the models and approaches used.

We have an additional repo focused on evaluation, perhaps it would be useful: https://github.com/microsoft/presidio-research
In addition, there are some research papers comparing Presidio to other approaches, but they usually use the off-the-shelf/predefined recognizers and do not customize the solution prior to evaluating it.

Hope it helps, if you look for anything more specific, please let us know.
